Traceback (most recent call last):
  File "C:\Users\Sotiris\anaconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\Sotiris\anaconda3\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\Sotiris\anaconda3\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\Sotiris\anaconda3\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\Sotiris\anaconda3\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\Users\Sotiris\anaconda3\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\Sotiris\anaconda3\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\Sotiris\anaconda3\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
The node/vertice set for this graph is $V = \{1,2,3,4,5\}$, and the edge set is $E=\{ \{1,2\}, \{1,3\}, \{1,4\}, \{2,4\}, \{2,5\}, \{3,4\} \}$. <br>
Alternatively, a graph may be specified by its [adjacency matrix](https://en.wikipedia.org/wiki/Adjacency_matrix), which is a square ($n \times n$) matrix with $0/1$ values representing the presence/absence of an edge between the row and column indices. For example, the graph above has adjacency matrix:

<img src="./img/2/adjacency_mat.svg" alt="adjacency matrix" style="width: 200px;"/>
------------------

[1;36m  Input [1;32mIn [2][1;36m[0m
[1;33m    The node/vertice set for this graph is $V = \{1,2,3,4,5\}$, and the edge set is $E=\{ \{1,2\}, \{1,3\}, \{1,4\}, \{2,4\}, \{2,5\}, \{3,4\} \}$. <br>[0m
[1;37m        ^[0m
[1;31mSyntaxError[0m[1;31m:[0m invalid syntax

SyntaxError: invalid syntax (752533824.py, line 1)

